frontend/src/Interface/FertilizerInterface.tsx

```diff
@@ -4,11 +4,11 @@ interface FertilizerInterface {
   fertilizerId: string;
   applRate: number;
   applUnitId: string;
-  applDate?: string;
-  applMethodId?: string;
-  customN: number;
-  customP2o5: number;
-  customK2o: number;
+  applDate: string;
+  applMethodId: string;
+  customN: number | null;
+  customP2o5: number | null;
+  customK2o: number | null;
```

Comment:
A question, what's the reason for null? if there's no custom it will be 0 anyways.

Reply:
nmp compatibility :smile: